### ISSUE TITLE
[TW-463] manpages should be installed from ${CMAKE_CURRENT_BINARY_DIR}

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#498]
_Jakub Wilk on 2013-02-26T14:19:16Z says:_ 

 Please see the attached patch. It fixes manpage installation for out-of-tree builds.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2013-02-26T14:19:16Z
Modified: 2014-02-09T01:54:32Z
```

**Comment by taskwarrior** [comment]
_Jakub Wilk on 2013-02-26T14:30:54Z says:_ 

 Bah. I just noticed "Fix: manpages were not installed when running an out-of-source build" commit, which supposedly also fixes the problem. I think that that commit  is incorrect (it defeats the purpose of out-of-tree builds) and should be undone.

**Comment by taskwarrior** [comment]
_Jakub Wilk on 2013-02-26T17:42:53Z says:_ 

 I realize by bug report isn't as clear as it should be. So let me try again.

* In taskwarrior 2.1.2, the manual pages are built into ${CMAKE_CURRENT_BINARY_DIR}, but then installed them from ${CMAKE_CURRENT_SOURCE_DIR}. This is obviously not going to work if the two variables differ, i.e. for out-of-tree builds.

* My patch (which is against 2.1.2, though I mistakenly rebased it against git HEAD) fixes this problem by installing manpages from ${CMAKE_CURRENT_BINARY_DIR}.

* Commit 9395a4d112722c063868b481664134b7ac18c9b4 ("Fix: manpages were not installed when running an out-of-source build") fixes the same problem differently: it builds manpages into ${CMAKE_CURRENT_SOURCE_DIR}. This sort of works, but the whole idea of out-of-tree builds is to put all the build artifacts into a separate directory...

So I propose to:
1) Undo 9395a4d112722c063868b481664134b7ac18c9b4.
2) Apply my patch instead.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2013-03-02T23:40:29Z says:_ 

 Applied.  Thank you.

**Comment by taskwarrior** [comment]
_Jakub Wilk on 2013-03-04T19:28:01Z says:_ 

 As I wrote in message #2, you need to undo commit 9395a4, too.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2013-03-05T01:24:24Z says:_ 

 Done.  Rather than apply the 9395a4.diff, I just ran:

<pre>
$ git revert 9395a4
</pre>

**Comment by pbeckingham** [comment]
[TW-463_fix-manpage-installation.diff.txt](https://github.com/GothenburgBitFactory/taskwarrior/files/1756128/TW-463_fix-manpage-installation.diff.txt)
[TW-463_undo-9395a4.diff.txt](https://github.com/GothenburgBitFactory/taskwarrior/files/1756129/TW-463_undo-9395a4.diff.txt)
